Application/src/Parser.java

```diff
@@ -0,0 +1,99 @@
+import Model.diagramObject;
+import com.google.gson.Gson;
+
+/**
+ * @author Rashad Kamsheh & Isabelle Törnqvist
+ * @since 2017-10-16
+ */
+
+public class Parser {
+
+    // odd number for the First Diagram counter
+    static int counter = 1;
+    // even number for the parallel Diagram counter
+    static int parallelCounter = 2;
+
+    static String Processes = "";
+    static String properProcesses = "";
+
+    static String FirstDiagram = "";
+    static String properFirstDiagram = "";
+
+    static String ParallelDiagram= "";
+    static String ProperParallelDiagram = "";
+
+
+    /**
+     * Parses the imported JSON files and prints it to the console
+     * @param inputJSON the imported collection of Strings that make up the JSON files
+     */
+
+    public static void Parser(String inputJSON){
```

Comment:
Why is the method called Parser for a class called Parser? 
Also why does the method start with AN UPPERCASE?? Like it's a class????

Reply:
good point